docs/index.md

```diff
@@ -1,241 +1,172 @@
----
+<!-- ---
```

Comment:
you don't want HTML quotes around the frontmatter, if you don't want to hide navigation, please remove it.